bin/circRNA_counts_matrix.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
```

Comment:
not for this PR, but in general can you add author and license please to your custom scripts?

Reply:
This script contains the requested information in lines 7-10. Maybe not in the expected location, but the only thing I did here was adding the shebang. Quickly went through all the other scripts, found author and license in various formats in all of them.

Should I unify the formatting of this information across all files?